paper-server/patches/sources/net/minecraft/world/level/block/entity/CreakingHeartBlockEntity.java.patch

```diff
@@ -0,0 +1,56 @@
+--- a/net/minecraft/world/level/block/entity/CreakingHeartBlockEntity.java
++++ b/net/minecraft/world/level/block/entity/CreakingHeartBlockEntity.java
+@@ -127,7 +_,7 @@
+                     if (creakingProtector.isPresent()) {
+                         Creaking creaking = creakingProtector.get();
+                         if (!CreakingHeartBlock.isNaturalNight(level) && !creaking.isPersistenceRequired()
+-                            || creakingHeart.distanceToCreaking() > 34.0
++                            || creakingHeart.distanceToCreaking() > creakingHeart.getDistanceCreakingTooFar() // Paper - custom distance for creaking
+                             || creaking.playerIsStuckInYou()) {
+                             creakingHeart.removeProtector(null);
+                         }
+@@ -199,9 +_,10 @@
+ 
+     @Nullable
+     public static Creaking spawnProtector(ServerLevel level, CreakingHeartBlockEntity creakingHeart) {
++        if (creakingHeart.getDistanceCreakingTooFar() <= 0) return null; // Paper - avoid spawn if the max distance is 0
```

Comment:
Is this even possible now with the codec?

Reply:
if allow the 0 then yes.